apps/cli/src/api.ts

```diff
@@ -1,35 +1,38 @@
-import { createApp } from './app';
-
+import { createApp } from "./app";
```

Comment:
NIT: I'm trying to keep 1 newline in between local imports (`./app`) and external (`path`) I think it helps us better see what external dependencies we need.

Reply:
This would mean that `./sys/AgentProtocolWorkspace` would move up to the top next to `./app`